server/index.js

```diff
@@ -119,14 +119,15 @@ export default class Server {
     }
   }
 
-  async start (port) {
+  async start (port, hostname) {
     await this.prepare()
     this.http = http.createServer(this.getRequestHandler())
     await new Promise((resolve, reject) => {
       // This code catches EADDRINUSE error if the port is already in use
       this.http.on('error', reject)
       this.http.on('listening', () => resolve())
-      this.http.listen(port)
+      if (hostname) this.http.listen(port, hostname)
```

Comment:
please make this a if/else with `{` and `}`, standardjs should have caught this 🤔